composer.json

```diff
@@ -19,10 +19,10 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
+        "php": "^8.1",
```

Comment:
better keep it low

Reply:
No, required for "nesk/rialto": "dev-zoon"